UserInterface/GameManager/GameManager.cs

```diff
@@ -12,35 +15,42 @@ public class GameManager : IGameManager
     {
         private readonly IUserInterface _userInterface;
         private readonly IWindow _window;
-        private readonly IAnimalActions _animalActions;
+        private readonly IObjectManager _objectManager;
         private readonly IPairManager _pairManager;
-        
+        private readonly IMovementManager _movementManager;
+
         private Board _board;
-        private List<Animal> _animals;
+        private List<IItem> _gameObjects;
         private List<Pair> _pairs;
-        private List<Animal> _children;
+        private List<IItem> _childrenGameObjects;
+
+        private Dictionary<ConsoleKey, GameObjectsInfo> _gameObjectsInfo = new Dictionary<ConsoleKey, GameObjectsInfo>();
 
         /// <summary>
         /// The class contains all logic layer to start a game from User Interface.
         /// </summary>
         /// <param name="userInterface">User Interface.</param>
         /// <param name="window">Window.</param>
-        /// <param name="animalActions">Animal Action.</param>
+        /// <param name="objectManager">Object manager.</param>
         /// <param name="pairManager">Pair manager.</param>
+        /// <param name="movementManager">Movement manager.</param>
         public GameManager(
             IUserInterface userInterface,
             IWindow window,
-            IAnimalActions animalActions,
-            IPairManager pairManager)
+            IObjectManager objectManager,
+            IPairManager pairManager,
+            IMovementManager movementManager)
         {
             _userInterface = userInterface;
             _window = window;
-            _animalActions = animalActions;
+            _objectManager = objectManager;
```

Comment:
Actually almost any variable containing word object should be regarded as illegal in C#, because in C# anything but null is object.

Rename all variables containing word object into something more descriptive.

Reply:
You changed _objectManager to _itemManager.
It is not better. Person without knowledge about your code will have no chance to guess what that means.

Reply:
Then what is better? GameUnitManager, GameEntityManager?